documentation/bin/createDocu.groovy

```diff
@@ -407,6 +527,8 @@ stepsDir = null
 stepsDocuDir = null
 String customDefaults = null
 
+stagesDocuDir = new File(Helper.projectRoot, "docs/stages")
```

Comment:
where is `stagesDocuDir` defined, is it a global variable?

Reply:
definition is done here ...